utils/dynamodb.ts

```diff
@@ -14,45 +12,49 @@ export const writeSignatureCheckpoint = (
 ) => {
   var params = {
     TableName: tableName,
-    // Key: {
-    //   earliest: { S: earliest },
-    //   latest: { S: latest },
-    // },
     Item: {
-      'id' : {S: 'CHECKPOINT'},
-      'earliest' : {S: earliest},
-      'latest': { S: latest }
-    }
+      id: { S: "CHECKPOINT" },
+      earliest: { S: earliest },
+      latest: { S: latest },
+    },
   };
 
   // Call DynamoDB to add the item to the table
   ddb.putItem(params, function (err, data) {
     if (err) {
       console.log("Error", err);
```

Comment:
should make this a console.warn or error too

Reply:
Good call